### PR TITLE
P2-496 Make plugin translateable

### PR DESCRIPTION
**Pull request by jdevalk**
## Summary

This PR can be summarized in the following changelog entry:

* Makes the plugin translateable.

## Relevant technical choices:

To make the plugin translateable and not have to write all the needed grunt stuff for it, I had to switch the plugin to using [@yoast/plugin-grunt-tasks](https://github.com/Yoast/plugin-grunt-tasks) more fully, which I've done.

This means:

* Quite a bit of grunt config was removed in favor of adding some configuration options to `package.json` and `Gruntfile.js`, so the grunt commands from `@yoast/plugin-grunt-tasks` could be used.

Other decisions:
* I've made all the PHP strings translateable, had to make some class variables into functions to allow for that.
* I've added a `.pot` file.
* I've introduced a `build` step which creates the `.pot` file and optimizes images (optimized one SVG in the same go).

## Milestone

* [x] I've attached the next release's milestone to this pull request.

## Test instructions

This PR can be tested by following these steps:

* Plugin should remain functional.
* All grunt commands should still work. 

Fixes P2-496

### CODE REVIEW

**Comment by johannadevos** [comment]
I just pushed a few commits in which I made more strings translatable, and added escaping. Whenever a translatable string is passed to `get_html`, `create_checkbox` or `create_select`, I didn't escape the string itself, since these methods (now) hold escape methods already. For all the other strings in the plugin, I added escaping.

**Comment by johannadevos** [comment]
CR: ✅ (on Joost's commits; I cannot review my own commits of course)
Acceptance: ✅ 

- [x] `grunt build` works properly.
- [x] I built a zip using `grunt artifact`. It holds all the files that are specified in the `copy` task.
- [x] With the zip, I did a smoke test that included resetting the plugin version and resetting the indexables. This functioned normally.

File: src/form-presenter.php [hunk 1]
```diff
@@ -47,7 +47,7 @@ public static function get_html( $title, $nonce_field, $fields, $submit = true )
 	public static function create_checkbox( $option, $label, $checked = false ) {
 		$checked_html = \checked( $checked, true, false );
 		$output       = \sprintf( '<input type="checkbox" ' . $checked_html . ' name="%1$s" id="%1$s"/>', $option );
-		$output      .= \sprintf( '<label for="%1$s">%2$s</label><br/>', $option, $label );
+		$output      .= \sprintf( '<label for="%1$s">%2$s</label><br/>', $option, \esc_html( $label ) );
```
[the remark this answers is not in the document]
Comment (hansjovis) [reply src/form-presenter.php]: Discussed this with @johannadevos , and she agrees. Fixing this myself 🙂.

**Review by hansjovis — APPROVED** [review]
Code review: 👍 

I did fix some minor issues (see commits).

**Pull request merged**